cinema/migrations/0002_movie_image.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 4.1 on 2024-12-16 06:03
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("cinema", "0001_initial"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="movie",
+            name="image",
+            field=models.ImageField(null=True, upload_to="???"),
```

Comment:
The `upload_to` parameter in the `ImageField` is currently set to `???`. You need to specify the correct path for image uploads. According to the task requirements, it should be something like `upload_to='images/'` or use a function like `movie_image_path` if you have defined one for dynamic paths.